vector/src/main/res/drawable/bg_carousel_page_1.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:endColor="#3372C7DA"
+        android:startColor="#33BBE7CF" />
+</shape>
```

Comment:
I do not have a strong opinion on it, but maybe move the new drawables to the module `ui-styles`? The webp images could stay in the `vector` project. IDK...

Reply:
sounds good to me :+1: 
https://github.com/vector-im/element-android/pull/4727/commits/785a14272040f7e3e0ab1ce205ab3782d0e12c41